homeassistant/components/ecovacs/__init__.py

```diff
@@ -56,35 +57,42 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up this integration using UI."""
 
     def get_devices() -> list[VacBot]:
-        ecovacs_api = EcoVacsAPI(
-            get_client_device_id(),
-            entry.data[CONF_USERNAME],
-            EcoVacsAPI.md5(entry.data[CONF_PASSWORD]),
-            entry.data[CONF_COUNTRY],
-            entry.data[CONF_CONTINENT],
-        )
-        ecovacs_devices = ecovacs_api.devices()
-        _LOGGER.debug("Ecobot devices: %s", ecovacs_devices)
-
-        devices: list[VacBot] = []
-        for device in ecovacs_devices:
-            _LOGGER.info(
-                "Discovered Ecovacs device on account: %s with nickname %s",
-                device.get("did"),
-                device.get("nick"),
-            )
-            vacbot = VacBot(
-                ecovacs_api.uid,
-                ecovacs_api.REALM,
-                ecovacs_api.resource,
-                ecovacs_api.user_access_token,
-                device,
+        try:
+            ecovacs_api = EcoVacsAPI(
+                get_client_device_id(),
+                entry.data[CONF_USERNAME],
+                EcoVacsAPI.md5(entry.data[CONF_PASSWORD]),
+                entry.data[CONF_COUNTRY],
                 entry.data[CONF_CONTINENT],
-                monitor=True,
             )
-
-            devices.append(vacbot)
-        return devices
+            ecovacs_devices = ecovacs_api.devices()
+            _LOGGER.debug("Ecobot devices: %s", ecovacs_devices)
+
+            devices: list[VacBot] = []
+            for device in ecovacs_devices:
+                _LOGGER.info(
```

Comment:
Do we need this at info level? Sounds like debug info.

Reply:
It was info level before, changed it to debug